addon/components/ember-attacher-inner.js

```diff
@@ -286,17 +289,17 @@ export default Ember.Component.extend({
   },
 
   _debouncedHideIfMouseOutsideTargetOrAttachment(event) {
-    Ember.run.debounce(this, this._hideIfMouseOutsideTargetOrAttachment, event, 10)
+    debounce(this, this._hideIfMouseOutsideTargetOrAttachment, event, 10);
   },
 
   _hideIfMouseOutsideTargetOrAttachment(event) {
     const target = this.get('_target');
 
     // If cursor is not on the attachment or target, hide the element
-     if (!target.contains(event.target)
-         && !(this.get('isOffset') && this._isCursorBetweenTargetAndAttachment(event))
-         // The ember-attacher-inner element is wrapped in the ember-attacher element
-         && !this.element.parentNode.contains(event.target)) {
+    if (!target.contains(event.target)
+      && !(this.get('isOffset') && this._isCursorBetweenTargetAndAttachment(event))
```

Comment:
I'd like to keep multi-line conditionals double-indented to show a clear distinction between the conditionals and inner block. Were you seeing eslint complain because of these?